workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/GetAppPage.tsx

```diff
@@ -4,19 +4,125 @@
  */
 
 import ClayButton from '@clayui/button';
-import {useState} from 'react';
-
-import {GetAppModal} from '../../components/GetAppModal/GetAppModal';
-
-export default function GetAppPage() {
-	const [showModal, setShowModal] = useState(false);
-
-	return (
-		<>
-			<ClayButton onClick={() => setShowModal(true)}>Get App</ClayButton>
-			{showModal && (
-				<GetAppModal handleClose={() => setShowModal(false)} />
-			)}
-		</>
-	);
-}
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { getSiteURL } from '../../components/InviteMemberModal/services';
+import { Liferay } from '../../liferay/liferay';
+import { getUrlParam } from '../../utils/getUrlParam';
+import AccountSelection from './components/AccountSelection';
+import ProductCard from './components/ProductCard';
+import { StepType } from './enums/stepType';
+
+type StepComponent = {
+  [key in StepType]?: JSX.Element;
+};
+
+type getAppProps = {
+  product?: Product;
+  selectedAccount?: Account;
+};
+
+const sectionProperties = {
+  [StepType.ACCOUNT]: {
+    backStep: StepType.ACCOUNT,
+    nextStep: StepType.LICENSES,
+    title: 'Account Selection',
+  },
+  [StepType.LICENSES]: {
+    backStep: StepType.ACCOUNT,
+    nextStep: StepType.PAYMENT,
+    title: 'License Selection',
+  },
+  [StepType.PAYMENT]: {
+    backStep: StepType.LICENSES,
+    nextStep: StepType.PAYMENT,
+    title: 'Payment Method',
+  },
+};
+
+const GetAPPFlow = () => {
```

Comment:
Please, change this name.
```suggestion
const GetAppFlow = () => {
```